app/i18n/cz/admin.php

```diff
@@ -168,8 +168,20 @@
 		'max-categories' => 'Maximální počet kategorií na uživatele',
 		'max-feeds' => 'Maximální počet kanálů na uživatele',
 		'registration' => array(
-			'help' => '0 znamená žádná omezení účtu',
 			'number' => 'Maximální počet účtů',
+			'select' => array(
+				'label' => 'Registration form',	// TODO - Translation
```

Comment:
@math-GH Replace them all with this and I think it should work.
```suggestion
				'label' => 'Registration form',	// TODO
```

Reply:
Actually better, use 

```diff
diff --git a/cli/i18n/I18nFile.php b/cli/i18n/I18nFile.php
index ef8f8776..16090bda 100644
--- a/cli/i18n/I18nFile.php
+++ b/cli/i18n/I18nFile.php
@@ -54,7 +54,7 @@ class I18nFile {
                $content = str_replace('<?php', '', $content);
 
                $content = preg_replace([
-                       "#',\s*//\s*TODO#i",
+                       "#',\s*//\s*TODO.*#i",
                        "#',\s*//\s*DIRTY#i",
                        "#',\s*//\s*IGNORE#i",
                ], [
```

Then you can just run `cli/manipulate.translation.php -a format` and it'll be automatically fixed.

Reply:
It is not enough. There is another problem. I am looking at it

Reply:
fixed the i18n files. Thanks for your help. Haven't seen it.

Reply:
I will make another PR with @Frenzie 's fix + a parse error catch

Reply:
https://github.com/FreshRSS/FreshRSS/pull/4197